snafu/hammerdb/hammerdb_wrapper.py

```diff
@@ -58,10 +56,6 @@ def __init__(self, parser):
         self.args.db_postgresql_storedprocs = ""
 
         # exporting the generic settings
-        if "es_server" in os.environ:
```

Comment:
do we need to define self.args.es_server from this still?

Reply:
As we discussed in slack, this is not required and I think it belongs to an old indexing implementation